api/src/types.rs

```diff
@@ -288,7 +288,7 @@ impl OutputPrintable {
 			OutputType::Transaction
 		};
 
-		let out_id = core::OutputIdentifier::from_output(&output);
+		let out_id = core::OutputIdentifier::from(*output);
```

Comment:
Maybe its better to change the `From<Output>` impl on `OutputIdentifier` to `From<&Output>`. This would avoid having to copy over the proof

Reply:
:+1:
Yes. I wanted to do that but could not remember what needed to be done to achieve this.
I was forgetting you could do `From<&Output>`.